modules/reactor/test/tests/meshgenerators/polygon_concentric_circle_mesh_generator/tests

```diff
@@ -85,171 +126,319 @@
+      detail = "duct_sizes and duct_intervals have different sizes."
+    []
+    [err_duct_sizes_length_2]
+      type = 'RunException'
+      input = 'poly_2d.i'
+      cli_args = 'Mesh/gen/duct_sizes="3.5 4.0" Mesh/gen/duct_intervals="2 2" Mesh/gen/duct_radial_biases="2"
+                  --mesh-only "poly_2d_err.e"'
+      expect_err = 'This parameter and duct_radial_biases must have the same length.'
+      detail = "duct_sizes and duct_radial_biases have different sizes."
+    []
+    [err_duct_block_ids]
+      type = 'RunException'
+      input = 'poly_2d.i'
+      cli_args = 'Mesh/gen/duct_sizes="3.5 4.0" Mesh/gen/duct_intervals="2 2"
+                  Mesh/gen/duct_block_ids="20"
+                  --mesh-only "poly_2d_err.e"'
+      expect_err = 'This parameter must have the same length as duct_intervals if set.'
+      detail = "duct_block_ids size is not consistent with number of ducts in the mesh."
+    []
+    [err_duct_block_names]
+      type = 'RunException'
+      input = 'poly_2d.i'
+      cli_args = 'Mesh/gen/duct_sizes="3.5 4.0" Mesh/gen/duct_intervals="2 2"
+                  Mesh/gen/duct_block_names="duct1"
+                  --mesh-only "poly_2d_err.e"'
+      expect_err = 'This parameter must have the same length as duct_intervals if set.'
+      detail = "duct_block_names size is not consistent with number of ducts in the mesh."
+    []
+    [err_descend_duct_sizes]
+      type = 'RunException'
+      input = 'poly_2d.i'
+      cli_args = 'Mesh/gen/duct_sizes="3.5 3.4" Mesh/gen/duct_intervals="2 2"
+                  --mesh-only "poly_2d_err.e"'
+      expect_err = 'This parameter must be strictly ascending.'
+      detail = "duct_sizes is not strictly ascending."
+    []
+    [err_undersize_duct_sizes]
+      type = 'RunException'
+      input = 'poly_2d.i'
+      cli_args = 'Mesh/gen/ring_radii="2 3" Mesh/gen/ring_intervals="2 2"
+                  Mesh/gen/duct_sizes="2.5 4.0" Mesh/gen/duct_intervals="2 2"
+                  --mesh-only "poly_2d_err.e"'
+      expect_err = 'This parameter must be positive and ensures no overlapping with rings.'
+      detail = "duct_sizes is non-positive or is low enough to be overlapped with rings."
+    []
+    [err_oversize_duct_sizes]
+      type = 'RunException'
+      input = 'poly_2d.i'
+      cli_args = 'Mesh/gen/duct_sizes="3.5 8.0" Mesh/gen/duct_intervals="2 2"
+                  --mesh-only "poly_2d_err.e"'
+      expect_err = 'This parameter must ensure that ducts are smaller than the polygon size.'
+      detail = "duct_sizes is bigger than the polygon."
+    []
+    [err_overlapped_background_boundary]
+      type = 'RunException'
+      input = 'poly_2d.i'
+      cli_args = 'Mesh/gen/background_inner_boundary_layer_width=5.0
+                  Mesh/gen/background_outer_boundary_layer_width=5.0
+                  --mesh-only "poly_2d_err.e"'
+      expect_err = 'must be less then the minimum thickness of the background region.'
```

Comment:
```suggestion
      expect_err = 'must be less than the minimum thickness of the background region.'
```
please fix the error message too. Sorry to be so nitpicky, I'm doing this as I m looking for other potential more important issues